peps/pep-0781.rst

```diff
@@ -0,0 +1,120 @@
+PEP: 781
+Title: Make ``TYPE_CHECKING`` builtin
```

Comment:
Grammar:
```suggestion
Title: Make ``TYPE_CHECKING`` a built-in variable
```

I think saying 'constant' is also fine here, as we mean the colloquial definition (very few things are truly constant in Python), but up to you:

```suggestion
Title: Make ``TYPE_CHECKING`` a built-in constant
```